sdk/storage/Azure.Storage.Blobs/src/Generated/Models/QueryFormat.cs

```diff
@@ -25,7 +25,7 @@ public QueryFormat(QueryFormatType type)
         public JsonTextConfigurationInternal JsonTextConfiguration { get; set; }
         /// <summary> Groups the settings used for formatting the response if the response should be Arrow formatted. </summary>
         public ArrowTextConfigurationInternal ArrowConfiguration { get; set; }
-        /// <summary> Any object. </summary>
+        /// <summary> parquet configuration. </summary>
```

Comment:
This file looks a bug fix to me.

Reply:
I agree.  Could you follow up on this?

Reply:
This is from M4 change. Previously it is generated as:
![image](https://user-images.githubusercontent.com/97225342/165206858-b9897e6e-8905-4611-85a0-c7fd30845394.png)
now it is:
![image](https://user-images.githubusercontent.com/97225342/165206808-bd258079-3e4f-49fa-bafe-a594095c1133.png)

Looking into its [swagger](https://raw.githubusercontent.com/Azure/azure-rest-api-specs/e432d9cc87bfed320d8feead4b448be9481c9181/specification/storage/data-plane/Microsoft.BlobStorage/preview/2021-04-10/blob.json). It is:
```
"ParquetConfiguration": {
    "xml": {
      "name": "ParquetTextConfiguration"
    },
    "description": "parquet configuration",
    "type": "object"
  }
```
So this change is by design to me.